lima.yaml

```diff
@@ -1,10 +1,8 @@
 images:
-  - location: https://cloud.debian.org/images/cloud/bookworm/20250814-2204/debian-12-genericcloud-amd64-20250814-2204.qcow2
+  - location: https://cloud.debian.org/images/cloud/bookworm/latest/debian-12-genericcloud-amd64.qcow2
```

Comment:
I feel like we should fix this for reproducibility reasons. You never know what might change